Task_DEV_4/Task_DEV_4/Circle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Circle : Figure
+    {
+        private const string errorMessage = "Radius cannot be less than 0.";
```

Comment:
ERROR_MESSAGE

Reply:
Need to insert one blank line between fields and properties.

Reply:
less or equal

Reply:
Done.